core/controllers/topic_editor_test.py

```diff
@@ -274,6 +274,15 @@ def test_access_topic_editor_page(self):
             expected_status_int=401)
         self.logout()
 
+        # Check that the editor page can not be accessed with an
+        # an invalid topic id.
+        self.login(self.NEW_USER_EMAIL)
+        self.get_html_response(
```

Comment:
Is it similar to test on line 255?

Reply:
The first test fails because there is no such topic. This one fails because the topic name is invalid.

Reply:
I see, thanks for the explanation. Can you please create a separate test for this check? (As suggested in the [wiki page](https://github.com/oppia/oppia/wiki/Writing-backend-tests#guidelines-for-writing-good-tests).)

Reply:
Done PTAL @DubeySandeep.